projects/Mallard/ios/Podfile.lock

```diff
@@ -506,4 +511,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 6f3fb3cc2db96d753c4cb301f1618796162d6d7c
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.9.1
```

Comment:
Might be an apple sign in thing but I think this change might just be because you've got a different version of cocoapods on your machine (think this is the latest) - might be worth doing:

```
gem uninstall cocoapods
gem install cocoapods -v 1.7.5
```

Reply:
FFS! I thought i fixed this lol